data/assets/nightsky/zenith.asset

```diff
@@ -0,0 +1,114 @@
+local earthAsset = asset.require('scene/solarsystem/planets/earth/earth')
+
+local textures = asset.syncedResource({
+  Name = "Planetarium Textures",
+  Type = "HttpSynchronization",
+  Identifier = "planetarium_textures",
+  Version = 1
+})
+
+
+local Position = {
+  Identifier = "ZenithPosition",
+  Parent = earthAsset.Earth.Identifier,
+  Transform = {
+    Translation = {
+      Type = "GlobeTranslation",
+      Globe = earthAsset.Earth.Identifier,
+      Latitude = 0,
+      Longitude = 0,
+      Altitude = 100000000,
+      UseCamera = true
+    },
+    Rotation = {
+      Type = "GlobeRotation",
+      Globe = earthAsset.Earth.Identifier,
+      Latitude = 0,
+      Longitude = 0,
+      UseHeightmap = false,
+      UseCamera = true
+    }
+  },
+  GUI = {
+    Name = "Zenith Position",
+    Path = "/Night Sky/Lines and Grids",
+    Hidden = false
+  }
+}
+
+local Plane = {
+  Identifier = "ZenithDot",
+  Parent = Position.Identifier,
+  Transform = {
+    Rotation = {
+      Type = "StaticRotation",
+      Rotation = {-1.57079, 90.0, 0.0}
```

Comment:
```suggestion
      Rotation = { -math.pi, 90.0, 0.0 }
```
I'm assuming the `90.0` is a mistake, since the rotation takes radians? So should be `math.pi / 4.0`